release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2471 @@
+xref:ocp-36-technology-preview[Technology Preview]) allows you to provision a
+service from the catalog.
+
+Select the desired service, then follow prompts for your desired project and
+configuration details.
+
+image::ocp36-add-from-catalog.gif[add from catalog]
+
+[[ocp-36-project-overview-redesign]]
+==== Project Overview Redesign
+
+In {product-title} 3.6, the Project Overview was resigned based on feedback from
+customers.
+
+In this redesign, there are three focused views:
+
+- Applications
+- Pipelines
+- Resource types
+
+There are now more contextual actions and rolled up metrics across multiple
+pods.
+
+image::ocp36-redesigned-project-overview.gif[Redesigned Project Overview]
+
+[[ocp-36-add-to-project]]
+==== Add to Project (Technology Preview)
+
+The add to project feature (currently in
+xref:ocp-36-technology-preview[Technology Preview]) allows you to provision a
+service without having to leave the Project Overview.
+
+When you go directly to the catalog from project, the context is preserved You
+can directly provision, then bind.
+
+image::ocp36-add-to-project.gif[add to project]
+
+[[ocp-36-bind-in-context]]
+==== Bind in Context
+
+The bind in context feature (currently in
+xref:ocp-36-technology-preview[Technology Preview]) allows you to provision a
+service and bind without having to leave the Project Overview.
+
+- Select deployment and initiate a bind.
+- Select from bindable services.
+- Binding is created and the user stays in context
+
+image::ocp36-bind-in-context.gif[bind in context]
+
+[[ocp-36-image-stream-details]]
+==== Image Stream Details
+
+In {product-title} 3.6, additional details are provided about image streams and
+their tags.
+
+This feature leverages Cockpit views from image streams. It details tags and
+provide information about each.
+
+image::ocp36-image-stream-details.png[bind in context]
+
+[[ocp-36-better-messages-for-syntax-errors]]
+==== Better Messages for Syntax Errors in JSON and YAML Files
+
+With {product-title} 3.6, better messages for syntax errors in JSON and YAML
+files are provided. This includes details of the syntax problem and the line
+number containing the issue.
+
+This feature validates input on commands such as `oc create -f foo.json` and
+`oc new-app -f template.yaml`. For example:
+
+----
+$ oc create -f dc.json
+error: json: line 27: invalid character 'y' looking   for beginning of value
+----
+
+[[ocp-36-cascading-deletes]]
+==== Cascading Deletes
+
+When deleting a resource, this feature ensures that all generated or dependent
+resources are also deleted.
+
+For example, when selecting a deployment configuration and deleting will delete
+the deployment configuration, deployment history, and any running pods.
+
+image::ocp36-cascading-deletes.png[cascading deletes]
+
+[[ocp-36-other-user-interface-changes]]
+==== Other User Interface Changes
+
+- {product-title} now shows relationships between bound applications and services
+in the Overview section.
```

Comment:
Suggest put this sentence `... shows relationships between ...` just at the end of former section `Bind in Context`. This looks more intuitive, because after `bind`, it will immediately `... shows relationships between ...`